src/macaron/output_reporter/results.py

```diff
@@ -83,6 +83,8 @@ class Record(Generic[RecordNode]):
         The context instance for this repo.
     dependencies : list[RecordNode]
         The list of Records for the analyzed dependencies of this repo.
+    policy_violation: bool
```

Comment:
I don't think this attribute is accessed/initialized anywhere in the code.

Reply:
Removed https://github.com/oracle-samples/macaron/pull/46/commits/4e61112595339f7871796e707f604cc1ff1b5711